docs/search/views/index.md

```diff
@@ -0,0 +1,317 @@
+
+Each View represents an inverted index that contains the search configuration and all document attributes in the linked collections. A Search View index consists of multiple segments that are each treated as a standalone index. Search Views are the only officially supported Views implementation, but you can create your own Search Views to further customize the way index and query data.
+
+The following table shows a comparison between C8Search Views and a full-text index:
+
+
+
+Feature                             | Search       | Full-text Index
+:-----------------------------------|:-------------|:---------------
+Term search                         | Yes          | Yes
+Prefix search                       | Yes          | Yes
+Boolean expressions                 | Yes          | Restricted
+Range search                        | Yes          | No
+Phrase search                       | Yes          | No
+Relevance ranking                   | Yes          | No
+Configurable Analyzers              | Yes          | No
+C8QL composable language construct  | Yes          | No
+Indexed attributes per collection   | Unlimited    | 1
+Indexed collections                 | Unlimited    | 1
+
+
+Views guarantee the best execution plan (merge join) when querying multiple attributes.
+
+## Concepts
+
+A View represents all documents available in a specified set of source collections. Each View is an abstraction of some transformation applied to documents in the collections. The type of transformation is specific to the View implementation and can be as simple as an identity transformation. 
+
+A Search View combines Boolean and generalized ranking retrieval and ranks each Boolean-approved document. For ranking text retrieval, we use the Vector Space Model (VSM) which uses documents and queries to represent vectors in a space formed by the *terms* of the query. A term can include single words, keywords, and phrases. You can use [Analyzers](#analyzers) to boost value analysis with tokenization.
+
+The document vectors that are closer to a query vector are more relevant. The closeness is expressed as the cosine of the angle between two vectors ([cosine similarity](https://en.wikipedia.org/wiki/Cosine_similarity)). We evaluate the following expression to define how relevant document `d` is to query `q`:
+
+`cos a = (d * q) / (|d| * |q|)`
+
+* `d * q` is the dot product of the query vector `q` and document vector `d`
+* `|d|` is the norm of vector `d`
+* `|q|` is the norm of vector `q`
+
+The vector components must be computed up front. Since space is formed by terms, you can use *term weights* as coordinates. The following probability and statistical weighting models are implemented in C8Search:
+
+* [Okapi BM25](https://en.wikipedia.org/wiki/Okapi_BM25)
+* [TF-IDF](https://en.wikipedia.org/wiki/Tf%E2%80%93idf)
+
+Both models rely on two components:
+
+* *Term frequency* (TF): The number of times term `t` appears in document `d`.
```

Comment:
Use underscores for italics, not asterisks. Also, this term should be bold, not italic. Same for the next one.

Reply:
Also, use hyphens rather than asterisks for the bullets. If you are using markdownlint, you should be able to change them all at once. (Highly recommend!)